libs/cli/package.json

```diff
@@ -9,9 +9,7 @@
 		"enquirer": "2.3.6",
 		"semver": "7.5.4"
 	},
-	"peerDependencies": {
-		"tslib": "^2.3.0"
-	},
+	"peerDependencies": {},
```

Comment:
At the minute Nx has an ESLint dependency check rule, which is complaining that this is an unused dependency.
I haven't yet verified which of these it has incorrectly removed - it may just be a case of adding this as an ignored dependency in the ESLint rule.